internal/pkg/api/schema.go

```diff
@@ -175,3 +175,47 @@ type StatusResponse struct {
 	Status  string                 `json:"status"`
 	Version *StatusResponseVersion `json:"version,omitempty"`
 }
+
+type FileInfo struct {
+	ActionID string     `json:"action_id"`
+	AgentID  string     `json:"agent_id"`
+	Source   string     `json:"src"`
+	File     FileData   `json:"file"`
+	Contents []FileData `json:"contents"`
+	Event    struct {
+		ID string `json:"id"`
+	} `json:"event"`
+	Host struct {
+		Hostname string `json:"hostname"`
+	} `json:"host"`
+}
+
+type FileData struct {
+	Size        int64  `json:"size"`
+	Name        string `json:"name"`
+	Extension   string `json:"ext"`
+	Mime        string `json:"mime_type"`
+	Compression string `json:"Compression"`
```

Comment:
(nit) Do we want a capitalized attribute here?

Reply:
most of these fields follow the ecs convention that lowercased fields represent an ecs standard, and initial-uppercase is reserved for custom fields not present in ECS.

Most of these fields are following the [ecs file](https://www.elastic.co/guide/en/ecs/current/ecs-file.html) standard. The uppercased are custom extensions, and are part of the [storage spec](https://docs.google.com/document/d/1BlCQYxHwgGZMxysc0BiWxT9ZCLKwgryhkD9ryDNDIuI/edit?usp=sharing). The capitalization needs to match for a Kibana App Services file plugin to read the known format